src/test.spec.ts

```diff
@@ -0,0 +1,62 @@
+///<reference path="../typings/index.d.ts" />
+declare var require: NodeRequire;
+
+import test = require('tape');
+import { EscapeStack } from './EscapeStack';
+
+const NUM_TIMES = 5;
+
+function addToStack(escapeStack: EscapeStack, t, numTimes: Number = 1) {
```

Comment:
Subtle gotcha here but in typescript, when referring to primitives, you shouldn't use their class name.  For example use these:

```
number
string
sometype[]
```

instead of

```
Number
String
Array<sometype>
```

Using the class name can sometimes cause weird compile errors that you wouldn't expect.